### DISCUSSION TITLE
能否增加用户黏性，添加签到送积分等功能

### DISCUSSION THREAD

**Discussion by jishuxiaowu2024** [halo-dev/halo#6487] [General]
### 描述一下此特性

能否增加用户黏性，添加签到送积分 会员等级经验等功能  同时添加付费阅读 可以发帖时选择为积分支付 我是wp框架过来的 想使用halo平替，但是目前halo不具备的功能太多 无法正常运营，希望开发团队可以更新，我愿意付费购买，如果愿意采纳可以联系qq1793290 我愿意配合

**Comment by ruibaby** [comment]
感谢反馈，目前 Halo 核心不考虑添加类似功能，但上述功能应该可以通过插件来满足需求。

> 我愿意付费购买，如果愿意采纳可以联系qq1793290 我愿意配合

商务问题建议查阅 https://www.halo.run/about 来联系我们。
